pkg/route/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type RouteSpec struct {
 	// be defaulted to Service.
 	To kapi.ObjectReference `json:"to"`
 
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []kapi.ObjectReference `json:"additionalTos,omitempty"`
```

Comment:
What's the point of AdditionalTos without weights?  If we want to expose alternates, the only reason to do so is to weight them.  Without weight, we might as well put the additional in annotations, because it's going to be just as painful to deal with.  I don't see users using AdditionalTo without weights, ever (unless I'm missing a use case)

So the comment from the previous PR stands

```
type RouteTargetReference struct {
  Kind string
  Name string
  Weight .... // either IntOrString or just int, depending on whether we argue this is relative or percentage based (relative is easier for mutation)
}

...
  To RouteTargetReference
  AdditionalTos []RouteTargetReference
...

```

I'm not in love with `AdditionalTos`, but we can change that at the end.